app/routes/germlineSmallMutation/export.download.js

```diff
@@ -131,8 +130,10 @@ router.get('/batch/download', async (req, res) => {
 
   // Prepare export
   const workbook = new Excel.Workbook();
+  // Get current date in format: YYYY-MM-DD
+  const date = new Date().toISOString().replace(/T(.*)/, '');
```

Comment:
Instead of using a regex here you could use `new Date().toLocaleString('en-ca', { year: 'numeric', month: 'numeric', day: 'numeric' })`

Reply:
I did noticed that there were other methods to get just the date without the time. However, I tried to maintain the format that was previously there (YYYY-MM-DD). For my code I maintain that format with todays date appearing as:
`2021-09-15`
I just ran your solution and did get the date, but it has a different format (DD/MM/YYYY). For todays date I got:
`9/15/2021`
Seeing as this is being used in the filename I would prefer not to have forward slashes in the title, but if you have another  approach that's native to node and is formatted like YYYY-MM-DD, I'm all ears.

Reply:
Was that with using `en-ca` as the locale? I just double checked and I'm getting the correct one using `en-ca`, but `en-us` gives the version with the slashes

Reply:
I copied the exact code from above
`new Date().toLocaleString('en-ca', { year: 'numeric', month: 'numeric', day: 'numeric' })`
It may be that we are using different versions of node to run this. I'm running this against node `v12` more specifically
`v12.20.2`
The same version that is currently running the API. Which version of node are you testing this on? If it's not the above version, can you try running it against the above version and let me know your results?

Reply:
Oh weird, that is the case. I tried it on playcode.io and assumed it was right as the docs on MDN also say the en-ca format is with dashes. Nevermind then

Reply:
I just tested your solution on Node `v14` and it works correctly. We can swap to your method when we switch to using node `v16` in October when the LTS version comes out and we remove support for Node `v12`.

Reply:
If you remember then sure. It's not a big deal though